sparkmagic/sparkmagic/kernels/kernelmagics.py

```diff
@@ -104,7 +104,7 @@ def help(self, line, cell="", local_ns=None):
   <tr>
     <td>sql</td>
     <td>%%sql -o tables -q<br/>SHOW TABLES</td>
-    <td>Executes a SQL query against the sqlContext.
+    <td>Executes a SQL query against the HiveContext(sqlContext) or SparkSession(spark).
```

Comment:
I would specify what possible names the context will have depending on which Spark version is being used.